codespell_lib/data/dictionary.txt

```diff
@@ -23922,12 +23929,14 @@ sepend->depend, suspend,
 sependent->dependent
 sepending->depending
 seperable->separable
+seperad->separate
```

Comment:
There is one hit on grep.app for seperadly->separately if you want to add that too.

Reply:
Added